docs/GUIDE.md

```diff
@@ -86,7 +106,7 @@ This command generates a graph and saves file named  `graph-of-<rule_id>-<date>.
 arf-to-graph -o ~/graphs scan-data/ssg-fedora-ds-arf.xml 'xccdf_org.ssgproject.content_rule_audit_rules_unsuccessful_\w+on_creat'
 ```
 
-This command generates a graph and saves file named `graph-of-<rule_id>-<date>.html` (The date the graph was created.) in `~/graphs` because `-o` was used. Then open web browser. _Default web browser is Firefox. If Firefox not installed it opens default web browser in OS._ 
+This command generates a graph and saves file named `graph-of-<rule_id>-<date>.html` (The date the graph was created.) in `~/graphs` because `-o` was used. Then open web browser. _Default web browser is Firefox. If Firefox not installed it opens default web browser in OS._
```

Comment:
```suggestion
This command generates a graph and saves file named `graph-of-<rule_id>-<date>.html` (The date the graph was created.) in `~/graphs` because `-o` was used. Then, it opens the generated file in your web browser. _Default web browser is Firefox. If Firefox is not installed, the default web browser in OS is used._
```